### PR TITLE
E UI 2333

### PR DESCRIPTION
**Pull request by ishOsw**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by Mo-Lala84 — APPROVED** [review]
Good work. Approved

File: src/shared/components/palette/text-area/write-text-area-field.component.ts [hunk 1]
```diff
@@ -13,4 +13,16 @@ export class WriteTextAreaFieldComponent extends AbstractFieldWriteComponent imp
   ngOnInit() {
     this.textareaControl = this.registerControl(new FormControl(this.caseField.value));
   }
+
+  isIE(): boolean {
+    return /msie\s|trident\/|edge\//i.test(window.navigator.userAgent);
+  }
+
+autoGrow(event) {
```
Comment (rayliangatos) [inline src/shared/components/palette/text-area/write-text-area-field.component.ts:21]: indent correctly?

**Review by rayliangatos — COMMENTED** [review]
@ishOsw it needs to upgrade the version of package.json, and RELEASE-NOTES.

**Review by rayliangatos — APPROVED** [review]

**Comment by issshiO** [comment]
> **Before creating a pull request make sure that:**
> 
> * [ ]  commit messages are meaningful and follow good commit message guidelines
> * [ ]  README and other documentation has been updated / added (if needed)
> * [ ]  tests have been updated / new tests has been added (if needed)
> 
> Please remove this line and everything above and fill the following sections:
> 
> ### JIRA link (if applicable)
>  EUI -2333 IE fix for textarea 
> **Does this PR introduce a breaking change?** (check one with "x")
> 
> ```
> [ ] Yes
> [ ] No
> ```

File: src/shared/components/palette/text-area/write-text-area-field.component.ts [hunk 2]
```diff
@@ -13,4 +13,17 @@ export class WriteTextAreaFieldComponent extends AbstractFieldWriteComponent imp
   ngOnInit() {
     this.textareaControl = this.registerControl(new FormControl(this.caseField.value));
   }
+
+  isIE(): boolean {
```
Comment (andywilkinshmcts) [inline src/shared/components/palette/text-area/write-text-area-field.component.ts:17]: Lee recently wrote a BrowserService that wrapped up this logic. Please check with him if it has been merged, and if so replace this function with a call to the BrowserService, which you will need to inject via the constructor

**Review by andywilkinshmcts — CHANGES_REQUESTED** [review]

File: src/shared/components/palette/text-area/write-text-area-field.component.ts (hunk 2 above)
Comment (issshiO) [reply src/shared/components/palette/text-area/write-text-area-field.component.ts:17]: > Lee recently wrote a BrowserService that wrapped up this logic. Please check with him if it has been merged, and if so replace this function with a call to the BrowserService, which you will need to inject via the constructor

yep. Done

Comment (ishOsw) [reply src/shared/components/palette/text-area/write-text-area-field.component.ts]: > Lee recently wrote a BrowserService that wrapped up this logic. Please check with him if it has been merged, and if so replace this function with a call to the BrowserService, which you will need to inject via the constructor

Yep. Done now

**Review by Mo-Lala84 — APPROVED** [review]
Approved

**Pull request merged**